app/Http/Controllers/EventController.php

```
@@ -18,336 +21,90 @@ public function index()
+        return view('events.show', compact('event', 'displayInstance'));
     }
 
     /**
      * Store a newly created resource in storage.
      */
-    public function store(Request $request)
+    public function store(StoreEventRequest $request, EventService $eventService)
     {
         $this->authorize('create', Event::class);
+    
+        $event = $eventService->createEventWithInstances(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$event'. (undefined)

(UnusedLocalVariable)